### ISSUE TITLE
Priority missing from write request

### ISSUE THREAD

**Issue by nickngsr** [fh1ch/node-bacstack#20]
Since the below change on EncodeWriteProperty all writes seem to fail
```javascript
if (priority === baAsn1.BACNET_NO_PRIORITY) {
    baAsn1.encodeContextUnsigned(buffer, 4, priority);
}
```
surely on most occasions this will be falsy and never apply the priority on the write request?

**Comment by fh1ch** [comment]
Hi @nickngsr 

Thanks for your bug report. This is highly appreciated 👍 

This is indeed a mistake  made during the last refactoring round and wasn't noticed due to missing test-coverage of `EncodeWriteProperty`. The fix will be available with the `0.0.1-beta.7` release and I will work on adding test coverage to those functions as well.

Cheers

**Comment by fh1ch** [comment]
Release `0.0.1-beta.7` available.
